examples/offline_inference_distributed_cont_batching.py

```diff
@@ -0,0 +1,78 @@
+"""
+This example shows how to use Ray Core for running offline batch inference
+distributively on a multi-nodes cluster. In contrast to the
+`offline_inference_distributed.py` example, you don't need to manually
+set the batch size.
+
+Learn more about Ray Core in https://docs.ray.io/en/latest/ray-core/walkthrough.html
+"""
+
+from typing import List
+
+import ray
```

Comment:
IIUC, you assume the Ray cluster has been started before running this example right? In this case it would be better to briefly comment in this script about how to start a Ray cluster with multiple nodes.